lib/Limitation/Mapper/NewObjectStateLimitationMapper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Limitation\Mapper;
+
+class NewObjectStateLimitationMapper extends ObjectStateLimitationMapper
```

Comment:
Cool, then the same goes for `ParentContentTypeLimitationMapper`, `ParentGroupLimitationMapper`, and `ParentOwnerLimitationMapper`. Checking...